container/go/cmd/loader/loader.go

```diff
@@ -33,17 +39,29 @@ var (
 	imgTarball = flag.String("tarball", "", "The path to the tarball to load.")
 )
 
+// Directory containing the pulled OCI image artifacts without modification.
+const ociImageDir = "image-oci"
+
+ // Directory containing actual images (symlinks or image.tar) to be used by container_import as the image.
```

Comment:
Directory containing an image and the image.tar that can be consumed by `container_import`